frontend/src/assets/locales/ro/translation.json

```diff
@@ -13,5 +13,17 @@
       "dashboard": "Dashboard"
     },
     "collapse": "Restrange meniul"
+  },
+  "organization_profile": {
+    "title": "Profil organizație",
+    "button_label": "Editează",
```

Comment:
This should be in "general". Please align with this PR: https://github.com/code4romania/teo/pull/15/files